viewing_party/party.py

```diff
@@ -1,23 +1,210 @@
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    '''
+    adds movie details to dictionary and returns movie details
+    if title, genre, or rating are None, function returns None
+    '''
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+
+    if title is None or genre is None or rating is None:
+        return None
+    elif title == False or genre == False or rating == False:
+        return None
+    else:
+        return new_movie
+
+def add_to_watched(user_data, movie):
+    '''
+    adds movie data in the form of a dictionary
+    to a dictionary of user_data, where the key is watched and
+    the values are a list of dictionaries of movies the user has watched
+    '''
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    '''
+    adds movie data in the form of a dictionary
+    to a dictionary of user_data, where the key is watchlist and
+    the values are a list of dictionaries of movies the user has queued to watch
+    '''
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    '''
+    moves a movie from watchlist to watched
+    '''
+    for movie in user_data["watchlist"]:
+        if title in movie.values():

```

Comment:
Consider doing it this way instead:
```suggestion
        if title == movie['title']:
```
This way we're clearer about exactly what we're checking. It also helps avoid weird edge case bugs (what if there was a movie whose _title_ was 'Horror'?)